Demo/Sources/Components/JobKeyInfo/JobKeyInfoDemoView.swift

```diff
@@ -0,0 +1,41 @@
+import FinniversKit
+
+class JobKeyInfoDemoView: UIView {
+
+    // MARK: - Private properties
+
+    private lazy var demoView = JobKeyInfoView(infoPairs: .default, withAutoLayout: true)
+
+    // MARK: - Init
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+
+    required init?(coder: NSCoder) { fatalError() }
```

Comment:
Just to follow same style, but not necessary: "fatalError("init(coder:) has not been implemented")"